GitHubContents.ps1

```diff
@@ -322,7 +325,6 @@ filter Set-GitHubContent
         [string] $CommitMessage,
 
         [Parameter(
```

Comment:
We may also want to allow this to be `ValueFromPipeline` to enable the content to be piped directly into this method.  That's certainly an optional change for this PR though.